pkg/ddc/jindofsx/transform.go

```diff
@@ -218,6 +222,11 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	// combine properties together
 	if len(runtime.Spec.Master.Properties) > 0 {
 		for k, v := range runtime.Spec.Master.Properties {
+			if k == "namespace.meta-dir" && runtime.Spec.Master.Replicas == 3 {
+				err = fmt.Errorf("Not Support Set NamespaceMetaDir With %v And Replicas = 3", v)
```

Comment:
err = fmt.Errorf("it's not able to support set namespaceMetaDir with %v and replicas = 3", v)

Reply:
done